repos/fdbt-site/src/pages/selectTimeRestrictions.tsx

```diff
@@ -136,19 +142,30 @@ const TimeRestrictionCard = ({ timeRestriction }: { timeRestriction: PremadeTime
                         </div>
                     </div>
 
-                    <TimeRestriction entity={timeRestriction} />
+                    <TimeRestrictionCardBody entity={timeRestriction} />
                 </div>
             </div>
         </div>
     );
 };
 
+export const isTimeRestrictionsDefinitionWithErrors = (
+    timeRestrictionsDefinition: TimeRestriction | TimeRestrictionsDefinitionWithErrors,
+): timeRestrictionsDefinition is TimeRestrictionsDefinitionWithErrors =>
+    (timeRestrictionsDefinition as TimeRestrictionsDefinitionWithErrors).errors !== undefined;
+
```

Comment:
Can we use the new style inline please:

"errors" in timeRestrictions ? timeRestrictions.errors : []